### PR TITLE
Saltstack has changed their label for from Low Hanging fruit to good …

### PR DESCRIPTION
**Pull request by pmmalinov01**
…first issue. So searches do not pop up

Please complete the following checklist if your PR is adding a new link to the list:

- [x] I've read [contributing guidelines](https://github.com/MunGell/awesome-for-beginners/blob/master/CONTRIBUTING.md)
- [x] This PR does not introduce duplicates to the list
- [ ] The link is added at the bottom of the list category
- [ ] Suggested repository is maintained, have an active community, is able to mentor new contributors and have issues with the suggested label
- [ ] The link I add follows the suggested pattern:

```
- [Repository Name](link-to-repository-label) _(label: beginner-friendly-label-in-the-repository)_ <br> Description
```

Example link formatting:

```
- [awesome-for-beginners](https://github.com/MunGell/awesome-for-beginners/labels/good-first-contribution) _(label: good-first-contribution)_ <br> A list of awesome beginners-friendly projects.
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -235,7 +235,7 @@ JSON Schema Based Editor. JSON Editor takes a JSON Schema and uses it to generat
 - [opsdroid](https://github.com/opsdroid/opsdroid/labels/good%20first%20issue) _(label: good first issue)_ <br> An open source chat-ops bot framework.
 - [pandas](https://github.com/pandas-dev/pandas/labels/good%20first%20issue) _(label: good first issue)_ <br> Flexible and powerful data analysis / manipulation library for Python, providing labeled data structures similar to R data.frame objects, statistical functions, and much more
 - [VPN@Home](https://github.com/ezaquarii/vpn-at-home/labels/good%20first%20issue) _(label: good first issue)_ <br> 1-click OpenVPN deployment and management dashboard.
-- [SaltStack](https://github.com/saltstack/salt/labels/Low-Hanging%20Fruit) _(label: Low-Hanging Fruit)_ <br> Software to automate the management and configuration of any infrastructure or application at scale.
+- [SaltStack](https://github.com/saltstack/salt/labels/Low-Hanging%20Fruit) _(label: good first issue)_ <br> Software to automate the management and configuration of any infrastructure or application at scale.
```
Comment (blackcat1383) [inline README.md:238]:
```suggestion
- [SaltStack](https://github.com/saltstack/salt/labels/good%20first%20issue) _(label: good first issue)_ <br> Software to automate the management and configuration of any infrastructure or application at scale.
```

Comment (pmmalinov01) [reply README.md]: Hi adding the %20 is a bit confusing for me. Can you tell why it is?

Comment (blackcat1383) [reply README.md]: Sure!
It's ASCII encode of " " (space).
U can find more here: https://www.w3schools.com/tags/ref_urlencode.ASP

**Comment by MunGell** [comment]
Thanks!

**Pull request merged**